### PR TITLE
Upgrade tf version to fix security warning

### PR DESCRIPTION
**Pull request by chicm-ms**
1. keras version need to be upgraded to work with tensorflow 1.15
2. keras accuracy key changed in 2.3.1

### CODE REVIEW

**Comment by ultmaster** [comment]
Are there tests (IT/UT) for these tuners?

**Comment by chicm-ms** [comment]
> Are there tests (IT/UT) for these tuners?

There is UT for network_morphism tuner, and ga_squad is a trial, no builtin tuner for ga_squad
